images/base-glibc-debian-bash/bin-sh-wrapper.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+bash "$@"
```

Comment:
This one does not use `BASH_ENV=/etc/profile` because the current tests in Dockerfile.test use `sh -lc ...` and `-l`/`--login` causes double activation.

IMO all `Dockerfile`s should use `BASH_ENV=/etc/profile` and the tests should be adapted to not use `-l`.